tests/py_runtime/py_runtime_tests.bzl

```diff
@@ -320,6 +320,32 @@ def _test_system_interpreter_must_be_absolute_impl(env, target):
 
 _tests.append(_test_system_interpreter_must_be_absolute)
 
+def _test_interpreter_sh_binary_target(name):
+    native.sh_binary(
```

Comment:
Rather than use sh_binary itself, lets create an executable rule that isn't well behaved (has multiple outputs).

I don't want to use sh_binary itself here because (1) we're interested in the behavior of multiple outputs, not sh_binary itself, and (2) I spent months cleaning up usages of py_* rules in tests because they incidentally had some behavior and they were convenient to use, but woo, that was a lot of work, so I don't want to propagate that bad practice.

Reply:
Done